src/org/thoughtcrime/securesms/ProfileActivity.java

```diff
@@ -97,17 +97,21 @@ protected void onCreate(Bundle bundle, boolean ready) {
 
     setSupportActionBar(this.toolbar);
     ActionBar supportActionBar = getSupportActionBar();
-    supportActionBar.setDisplayHomeAsUpEnabled(false);
-    supportActionBar.setCustomView(R.layout.conversation_title_view);
-    supportActionBar.setDisplayShowCustomEnabled(true);
-    supportActionBar.setDisplayShowTitleEnabled(false);
-    Toolbar parent = (Toolbar) supportActionBar.getCustomView().getParent();
-    parent.setPadding(0,0,0,0);
-    parent.setContentInsetsAbsolute(0,0);
-
-    titleView = (ConversationTitleView) supportActionBar.getCustomView();
-    titleView.setOnBackClickedListener(view -> onBackPressed());
-    titleView.setOnClickListener(view -> onEnlargeAvatar());
+    if (isGlobalProfile()) {
+      supportActionBar.setDisplayHomeAsUpEnabled(true);
+      supportActionBar.setHomeActionContentDescription(getString(R.string.back));
```

Comment:
is that really needed? i would expect that the default text just reads "back". at least we are not doing that at all the other placed where we set the back-button.

maybe better remove this line here - and if we really think, it is important, do a separate pr and set that at all places.

Reply:
it is needed because android sets "navigate up" as content description, the other back buttons are part of our custom view